ui/app/AppLayouts/Onboarding/views/ConfirmPasswordView.qml

```diff
@@ -0,0 +1,140 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.12
+
+import shared.controls 1.0
+import shared 1.0
+import shared.panels 1.0
+import utils 1.0
+
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+
+import "../stores"
+
+Page {
+    id: root
+
+    property string password
+
+    signal backClicked()
+
+    anchors.fill: parent
+
+    Component.onCompleted: confPswInput.forceActiveFocus(Qt.MouseFocusReason)
+
+    Column {
+        id: view
+        spacing: 4 * Style.current.padding
+        width: 416
+        anchors.centerIn: parent
+
+        StatusBaseText {
+            anchors.horizontalCenter: parent.horizontalCenter
+            text: qsTr("Have you written down your password?")
+            font.pixelSize: 22
+            font.bold: true
+            color: Theme.palette.directColor1
+        }
+
+        Column {
```

Comment:
Since StatusBaseText is a Text type, could it be that we use [lineHeight](https://doc.qt.io/qt-5/qml-qtquick-text.html#lineHeight-prop) instead of wrapping several Text components in a Column?